<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Employee</class>
 <widget class="QMainWindow" name="Employee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_close">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>خـــروج</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>441</width>
      <height>601</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_leave_4">
     <attribute name="title">
      <string>درخــــواســـت مـــرخــصـــی</string>
     </attribute>
     <widget class="QTextBrowser" name="text_qu_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>391</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>System</family>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'System'; font-size:7.8pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p align=&quot;right&quot; dir='rtl' style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:16pt; font-weight:400; vertical-align:super;&quot;&gt;لطفا جهت ثبت در خواست مرخصی کادر زیر وارد کنید.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_leave_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>411</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>برگه درخواست مرخصی</string>
      </property>
      <widget class="QTextBrowser" name="text_qu_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>391</width>
         <height>131</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>System</family>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'System'; font-size:7.8pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p align=&quot;right&quot; dir='rtl' style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:16pt; font-weight:400; vertical-align:super;&quot;&gt;اینجانب                                    &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;right&quot; dir='rtl' style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:16pt; font-weight:400; vertical-align:super;&quot;&gt;درخواست                         ساعت مرخصی را دارم.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="employee_firstname">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>40</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="leave_hour_4">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>70</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_apply_leave_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>تایید</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="employee_lastname">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_overtime_4">
     <attribute name="title">
      <string>درخـــواســـت اضــافــه کـــار</string>
     </attribute>
     <widget class="QTextBrowser" name="text_qu_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>391</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>System</family>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'System'; font-size:7.8pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p align=&quot;right&quot; dir='rtl' style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:16pt; font-weight:400; vertical-align:super;&quot;&gt;لطفا جهت ثبت در خواست اضافه کار کادر زیر وارد کنید.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_overtime_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>411</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>برگه درخواست اضافه کار</string>
      </property>
      <widget class="QTextBrowser" name="text_qu_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>391</width>
         <height>131</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>System</family>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'System'; font-size:7.8pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p align=&quot;right&quot; dir='rtl' style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:16pt; font-weight:400; vertical-align:super;&quot;&gt;اینجانب                                    &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;right&quot; dir='rtl' style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:16pt; font-weight:400; vertical-align:super;&quot;&gt;درخواست                         ساعت اضافه کار را دارم.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="employee_firstname_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>40</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="overtime_hour_4">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>70</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_apply_overtime_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>تایید</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="employee_lastname_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_showCustomerInfo_4">
     <attribute name="title">
      <string>نمـــایش اطلاعات مـــشتــری</string>
     </attribute>
     <widget class="QLineEdit" name="cumtomer_natinal_code_show_info_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>کد ملی مشتری:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_show_customer_info_4">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>تایید</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="customer_info_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>341</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_showPersonalInfo_4">
     <attribute name="title">
      <string>نمایش اطـــلاعـــات شـــخصی</string>
     </attribute>
     <widget class="QTextEdit" name="employee_info_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>371</width>
        <height>511</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <kerning>false</kerning>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_openAccount_4">
     <attribute name="title">
      <string>باز کردن حساب برای مشتری</string>
     </attribute>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>کد ملی مشتری:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cumtomer_natinal_code_open_account_7">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_open_account_4">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>250</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>تایید</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cumtomer_money_for_open_account_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>90</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>مبلغ ابتدایی:</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="lable_13">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>371</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Zilla Slab Light</family>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Zilla Slab Light'; font-size:7.8pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p align=&quot;justify&quot; dir='rtl' style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt; font-weight:400; vertical-align:super;&quot;&gt;توجه!!!&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; dir='rtl' style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt; font-weight:400; vertical-align:super;&quot;&gt; مبلغ واریزی ابتدایی ، جهت افتتحاح حساب باید حداقل&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;justify&quot; dir='rtl' style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt; font-weight:400; vertical-align:super;&quot;&gt; 000 50 تومان باشد.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_deleteAccount_4">
     <attribute name="title">
      <string>حــــــذف حساب یـک مـشتری</string>
     </attribute>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>کد ملی مشتری:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cumtomer_natinal_code_delete_account_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_delete_account_4">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>150</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>تایید</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>90</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string> شماره حساب:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="account_num_delete_account_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>Employee</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>673</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>661</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
